src/extension/gogoduck/README.md

```diff
@@ -0,0 +1,76 @@
+# GoGoDuck
+
+GoGoDuck is a simple NetCDF file aggregator, using nco tools (ncks, ncrcat, ncatted, etc).
+
+## Requirements
+
+ * nco tools, a recent version! (>= 4.3.4)
+ * java/maven
+
+## Usage:
+
+To build:
+```
+$ mvn clean install
+```
```

Comment:
Looks like they moved some dependencies to a geotools repository - I had to make changes to the geoserver-build project to pull them in now